code/index.html

```diff
@@ -3,20 +3,137 @@
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
+      <hr>
+      <section class="other-news">
+        <div class="small-news">
+          <img class="image-other" 
+          src="./biscuit-basket.jpg" 
+          alt="Biscuit in a basket">
+          <div class="text-other-news">
+            <a href="#"> 
+              <h4>Kex/tjex argument continues in Swedish parlament
+              </h4>
+            </a>
+          </div> 
+        </div>
+        <div class="small-news">
+          <img class="image-other" 
+          src="./jam-biscuits.jpg" 
+          alt="Biscuits in a box">
+          <div class="Jam biscuits">
+            <a href="#">
+              <h4>New version of the Singoalla biscuit in stores next month
+              </h4>
+            </a>
+         </div>
+        </div>
+        <div class="small-news">
+          <img class="image-other" 
+          src="./canoe.jpg" 
+          alt="Picture of a woman in a canoe">
+          <div class="text-other-news">
+            <a href="#">
+              <h4>Fika for adventurers; tips on what to bring
+              </h4>
+            </a>
+          </div>
+        </div>
+        <div class="small-news">
+          <img class="image-other" 
+          src="./child-baking.jpg" 
+          alt="Children baking">
+          <div class="text-other-news">
+            <a href="#">
+                <h4>How to get you child interested in baking
+                </h4>
+              </a>
+          </div>
+        </div>
+      <div class="small-news">
+        <img class="image-other" 
+        src="./cinnamon-bun.jpg" 
+        alt="Cinnamon buns">
+        <div class="text-other-news">
+          <a href="#">
+            <h4>Research indicates that fika could be the solution to world peace
+            </h4>
+          </a>
+        </div>
+      </div>
+      <div class="small-news">
+        <img class="image-other" 
+        src="./coffee-cups.jpg" 
+        alt="Cups of coffee">
+        <div class="text-other-news">
+          <a href="#">
+            <h4>What your coffee cup tells us about you
+            </h4>
+          </a>
+        </div>
+      </div>
+      <div class="small-news">
+        <img class="image-other" 
+        src="./cake-fruit.jpg" 
+        alt="Cakes with fruit">
+          <div class="text-other-news">
+            <a href="#">
+              <h4>If there's fruit on them, they're healthy!
+              </h4>
+           </a>
+         </div>
+      </div>
+      <div class="small-news">
+        <img class="image-other" 
+        src="./flower-cakes.jpg" 
+        alt="Cakes with piped flowers">
+        <div class="text-other-news">
+          <a href="#">
+            <h4>How your choice in wedding fika can make your special day unique
+            </h4>
+          </a>
+        </div>
+      </div>
     </section>
+  </section>
   </body>
 </html>
```

Comment:
No questions regarding the html, I would suggest adding some more spacing between the code to make it a bit easier to read.